contracts/KyberIEOWrapper.sol

```diff
@@ -1,7 +1,57 @@
-pragma solidity ^0.4.0;
+pragma solidity ^0.4.23;
 
-contract KyberIEOWrapper {
-    function KyberIEOWrapper(){
 
+import './KyberIEO.sol';
+import './ERC20Interface.sol';
+
+
+interface KyberNetwork {
+    function trade(ERC20 src, uint srcAmount, ERC20 dest, address destAddress, uint maxDestAmount,
+        uint minConversionRate, address walletId) external payable returns(uint);
+    function getExpectedRate(ERC20 src, ERC20 dest, uint srcQty) external view
+        returns (uint expectedRate, uint slippageRate);
+}
+
+
+contract KyberIEOWrapper is Withdrawable {
+
+    KyberNetwork kyberNetwork;
+    KyberIEO kyberIeo;
```

Comment:
where do you init it? in any case should be part of the contrinute params.

Reply:
fixed